pkg/cloud/api/diff.go

```diff
@@ -55,10 +67,11 @@ func (r *DiffResult) add(state DiffItemState, p Path, a, b reflect.Value) {
 		Path:  p,
 	}
 	if a.IsValid() {
-		di.A = a.Interface()
+		di.A = a.String()
+
 	}
 	if b.IsValid() {
-		di.B = b.Interface()
+		di.B = b.String()
```

Comment:
See comment above, I'm not sure this should be changed.

Reply:
as above